website/content/docs/v0.9/Guides/troubleshooting-control-plane.md

```diff
@@ -0,0 +1,440 @@
+---
+title: "Troubleshooting Control Plane"
+description: "Troubleshoot control plane failures for running cluster and bootstrap process."
+---
+
+This guide is written as series of topics and detailed answers for each topic.
+It starts with basics of control plane and goes into Talos specifics.
+
+This document mostly applies only to Talos 0.9 control plane based on static pods.
+If Talos was upgraded from version 0.8, it might be still running self-hosted control plane, current status can
+be checked with the command `talosctl get bootstrapstatus`:
+
+```bash
+$ talosctl -n IP get bs
+NODE         NAMESPACE   TYPE              ID              VERSION   SELF HOSTED
+172.20.0.2   runtime     BootstrapStatus   control-plane   1         false
+```
+
+In this guide we assume that Talos client config is available and Talos API access is available.
+Kubernetes client configuration can be pulled from control plane nodes with `talosctl -n IP kubeconfig`
+(this command works before Kubernetes is fully booted).
+
+### What is a control plane node
```

Comment:
```suggestion
### What is a control plane node?
```

Reply:
Feels like all of these headers should have question marks after them.

Reply:
our linter doesn't allow punctuation in headers, but I could suppress the warning, I like with `?` better

Reply:
Well, maybe not all of them, but the ones that are written as questions.